mmd/connconfig.go

```diff
@@ -15,11 +16,13 @@ type ConnConfig struct {
 	Url               string
 	ReadSz            int
 	WriteSz           int
+	Counter           int32
```

Comment:
per offline, remove `er`, and maybe rename (this name is Y.A.S. (yet another suggestion, open to other ideas too))
```suggestion
	LastConnVersion           int32
```